specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json

```diff
@@ -0,0 +1,594 @@
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SettingsName"
+          },
+          {
+            "$ref": "#/parameters/Settings"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Settings"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/settings/onboard": {
+      "post": {
+        "x-ms-examples": {
+          "Onboard workspace": {
+            "$ref": "./examples/settings/Onboard.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Onboards workspace to sentinel.",
+        "operationId": "OnboardWorkspace",
+        "parameters": [
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/OnboardingRequest"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OnboardingRequest"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/settings/offboard": {
+      "post": {
+        "x-ms-examples": {
+          "Onboard workspace": {
+            "$ref": "./examples/settings/Offboard.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Offboards workspace from sentinel.",
+        "operationId": "OffboardWorkspace",
```

Comment:
Should follow Noun_Verb format. Suggest `Workspace_Offboard` and `Workspace_Onboard`